core/src/commonMain/kotlin/ua/syt0r/kanji/presentation/common/resources/string/EnglishStrings.kt

```diff
@@ -458,6 +458,13 @@ object EnglishWritingPracticeStrings : WritingPracticeStrings {
     override val leftHandedModeMessage: String =
         "Adjusts position of input in landscape mode of writing practice screen"
 
+    override val strokesMessage: (count: Int) -> String = {
+        buildString {
+            append(it.toString())
+            if (it == 1) append(" stroke")
+            else append(" strokes")
```

Comment:
(Far from a kotlin expert, but the standard way of doing this would be the `plurals` resource like this:
```
    <plurals name="writing_practice_summary_mistakes">
        <item quantity="one">%d mistake</item>
        <item quantity="other">%d mistakes</item>
    </plurals>
```

With that said we already have something like this function precisely for mistakes in another place:
```
    override val savingMistakesMessage: (count: Int) -> String = {
        if (it == 1) "1 mistake" else "$it mistakes"
    }
```
So it probably doesn't really matter either way)

Reply:
Yeah, I'm not using resources because it's a multiplatform project and I didn't want to introduce some 3rd party library since it can be done this way anyway